### DISCUSSION TITLE
Quais tecnologias serão usadas no microserviços

### DISCUSSION THREAD

**Discussion by BravusDeveloper** [cleyton1986/microsservices-delphi#7] [Q&A] [answered]
Quais as tecnologias serão usadas no projeto e fazer o funcionar esse ecossistema de microserviços?

**Comment by cleyton1986** [comment, marked as answer]
Neste projeto serão utilizadas as tecnologias:

- [x] Delphi como linguagem/ferramenta
- [x] Horse como web framework
- [x] Redis como banco de cache
- [x] RebbitMQ como mensageria
- [x] Outros
